Code/Tools/AssetProcessor/Platform/Linux/AssetProcessor_Traits_Linux.h

```diff
@@ -7,5 +7,13 @@
  */
 #pragma once
 
+#include <climits>
+
+#ifndef PATH_MAX
+// this shouldn't happen
+#define PATH_MAX 260
+#endif
```

Comment:
If anything we should have this default to 4096 on Linux and MacOS in case PATH_MAX is not defined in either the `<limits>` or `<sys/syslimits.h>` header.

Reply:
`255` is the maximum filename length, where `4096` is the maximum path length. I think it should be `4096` for Linux and Mac

Reply:
set the max length to 4096 as a fallback and added a compile time error message